app/main.ipynb

```diff
@@ -192,17 +236,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": null,
    "outputs": [],
    "source": [
-    "# write your code here"
+    "mean_by_x = A.mean(axis=1)"
```

Comment:
incorrect result, you have 2 columns, but got 5 values, weird, isn't it?)
fix everywhere, here is a tip
![image](https://github.com/mate-academy/py-numpy/assets/61380547/9a013452-43bc-40a5-808e-e1270a4a3ae4)